testdata/includes_remote/Taskfile.yml

```
@@ -0,0 +1,4 @@
+version: '3'
+
+includes:
+  first: "{{ .FIRST_REMOTE_URL }}"
```

Missing ending line break. Also, removing spaces to adhere to convention.

```suggestion
  first: "{{.FIRST_REMOTE_URL}}"
```